ansible/roles/sysbench/tasks/main.yml

```diff
@@ -81,7 +81,7 @@
   changed_when: false
 
 - name: Set up VTGate Loadbalancing
-  shell: iptables -t nat -A OUTPUT -p tcp --dport 3306 -m state --state NEW -m statistic --mode nth --every {{ ((gateways | length) - index) }} --packet 0 -j DNAT --to-destination {{ gateway }}
+  shell: iptables -t nat -A OUTPUT -p tcp --dport 13306 -m state --state NEW -m statistic --mode nth --every {{ ((gateways | length) - index) }} --packet 0 -j DNAT --to-destination {{ gateway }}
```

Comment:
Yes. I changed the default port that points to vtgate's load balancer (from `3306` to `13306`) because all tcp packets with port `3306` were routed towards one of the vtgates. However, we not only want to communicate with vtgates,  we also want to communicate with other MySQL hosts, for instance: the SQL database used to store results.

This is why the configuration file contains the variable shown below, it allows us to declare the port of our vtgate load balancer.

```yaml
macrobench_all_mysql-port: 13306
```